test/regress/regress2/sygus/issue4022-conjecture-gen.smt2

```diff
@@ -1,3 +1,5 @@
+; COMMAND-LINE: -q
```

Comment:
one day we should grep `-q` in the regression tests and try to eliminate them, like we did once for `--no-check-proof`. But I guess that day is not here yet, and now we are still adding `-q`s :)

Reply:
Yep, exactly, I'm planning to fix check-models soon. The core issue is that bodies of quantified formulas are not evaluated wrt substitutions when they should be.